src/app/components/forms/financials/managed-account/managed-account.vue

```diff
@@ -316,7 +326,7 @@ export default class FormFinancialsManagedAccount extends mixins(Wrapper) implem
 
 	get isVerificationPending() {
 		// If they're in pending state and we don't require more info from them.
-		if (!this.account || this.account.status !== 'pending') {
+		if (!this.account || this.account.status !== this.PendingStatus) {
```

Comment:
outside of templates, use the enum directly instead of the one on the class instance. it'll be easier to convert these components in the future then